assets/js/popover.js

```diff
@@ -7,6 +7,7 @@ function htmlToElement(html) {
 
 function initPopover(baseURL, useContextualBacklinks, renderLatex) {
   const basePath = baseURL.replace(window.location.origin, "")
+  const regex = /\[!.+\]-? /;
```

Comment:
can we move the regex replacement function to `removeMarkdown()`?

Reply:
Moved